### PR TITLE
gtk: use "assertion" = "skip" instead of manual implementation

### PR DESCRIPTION
**Pull request by Cogitri**
GTK3 version of https://github.com/gtk-rs/gtk4-rs/pull/198

### CODE REVIEW

**Comment by Cogitri** [comment]
Also contains https://github.com/gtk-rs/gtk4-rs/pull/196, figured a seperate PR for 4 lines wouldn't make much sense :)

**Review by sdroege — APPROVED** [review]

File: gtk/src/auto/functions.rs [hunk 1]
```diff
@@ -233,7 +249,7 @@ pub fn device_grab_remove<P: IsA<Widget>>(widget: &P, device: &gdk::Device) {
 
 #[doc(alias = "gtk_disable_setlocale")]
 pub fn disable_setlocale() {
-    assert_initialized_main_thread!();
+    skip_assert_initialized!();
```
Comment (sdroege) [inline gtk/src/auto/functions.rs:252]: Shouldn't this be `assert_uninitialized`?

Comment (Cogitri) [reply gtk/src/auto/functions.rs:252]: Oh derp, too much copy pasting in the config, good catch :)

Comment (Cogitri) [reply gtk/src/auto/functions.rs]: Fixed, thanks

**Review by sdroege — APPROVED** [review]

**Review by GuillaumeGomez — APPROVED** [review]

**Pull request merged**